components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpointTest.java

```diff
@@ -81,7 +81,7 @@ public class JwksEndpointTest extends PowerMockIdentityBaseTest {
     private static final String CERT_THUMB_PRINT = "generatedCertThrumbPrint";
     private static final String ALG = "RS256";
     private static final String USE = "sig";
-    private static final JSONArray X5CJA = new JSONArray();
+    private static final JSONArray X5C_ARRAY = new JSONArray();
```

Comment:
why it is static final? we are modifying this arry in the class

Reply:
As this is a final reference variable, adding final keyword does not limit us from modifying the properties of the variable. But it does not allow us to reassign it.